bit/utils.py

```diff
@@ -104,16 +104,49 @@ def get_signatures_from_script(script):
     :returns: A list of retrieved signature from the provided scriptSig.
     :rtype: A ``list`` of ``bytes`` signatures
     """
+
+    def is_signature(sig):
```

Comment:
Let's define this outside of a function. Also, is it possible to use https://github.com/ofek/coincurve?

Reply:
Uh, great! I didn't find a function in coincurve dedicated for checking DER-encoding of signatures. However, it seems like we can "misuse" the function `coincurve.ecdsa.der_to_cdata`, which will produce a ValueError if the passed argument is **not** a DER-encoding of a signature. I have added a commit with the changes.